config/Configured.sol

```diff
@@ -38,11 +38,11 @@ contract Configured is StdChains {
         }
     }
 
-    function _txNameRawData() internal view virtual returns (string memory) {
+    function _txNameTransactionData() internal view virtual returns (string memory) {
```

Comment:
```suggestion
    function _txName() internal view virtual returns (string memory) {
```
The naming was ok but you just had to switch them up (`txName` -> `txNameRawData`, `txNameRawData` -> `txName`) because what you called raw data actually was the tx data.

I am ok with naming them `txNameRawData` & `txNameTransactionData` (`txNameTxData` for short)